src/Symfony/Component/Workflow/TransitionBlockerList.php

```diff
@@ -63,7 +63,7 @@ public function isEmpty(): bool
      *
      * @return \ArrayIterator|TransitionBlocker[]
      */
-    public function getIterator()
+    public function getIterator(): \ArrayIterator
```

Comment:
That should typehint \Traversable. That it returns an \ArrayIterator is an implementation detail that people should not rely on. Please also add the return type to `GlobResource::getIterator`. Then it's good to be merged IMO.